_ont_xgs/ont-fs-XGS-ONU-25-20NI.md

```diff
@@ -0,0 +1,586 @@
+
+#ONT/system/mib>
+```
+
+### Dump of a MIB
+Output a dump of mib to `/tmp/mibdump.txt`
+
+```
+#ONT/system/mib> dump
+MIB is saved to the file /tmp/mibdump.txt.
+#ONT/system/mib>
+```
+
+
+## Getting Speed LAN Mode
+Speed LAN Mode can check it from the `Status` column displayed by the following command.
+```
+#ONT>traffic/eth/show pack
+
+build time Apr 26 2023: 06:23:15
+
+-------------------- Line Pack -- PWR 0X0      --
+Line Slot 10, NumOfPorts 1, type 49, subtype 75, state 2
+-------------------- Configuration -------------------
+port | enable | loop | Mode | RL Type | RL Us Rate| RL Ds Rate|
+1       Yes     No      Auto            0       0       0
+
+-------------------- Status -------------------
+1) link state: Up, link mode: 2.5G Full
+
+-------------------- Bridge Pack -------------------
+Type:       75
+State:      2
+MAC Table:  0
+MAC Aging:  0
+MTU:        0
+-------------------- Chip Data -------------------
+eth Fd:         7
+port Mask:      0x0018
+type            28
+
+********** DEBUG INFO **********
+                 EmrLogId  : 5
+               EmrAdminEn  : 0
+              EmrbeInited  : 1
+              EmrDumpConn  : 0
+             EmrPwrShedEn  : 0x0
+           EmrConnItemNum  : 0
+         EmrUsDsReverseEn  : 1
+       EmrConnAllVidCheck  : 0
+      EmrMcastGemVlanOpEn  : 1
+       gEmrSaveConnItemEn  : 1
+      gEmrMcastCrossVlanEn : 1
+    EmrUniExtractPriMatch  : 1
+ EMR_DRV_PACK_DUMP_ITEM_EN : 0
+********************************
+#ONT>
+```
+
+
+# GPON/OMCI settings
+XGS-ONU-25-20NI ONT uses scfg.txt file and misc command for configuration.
+### scfg.txt
+In the XGS-ONU-25-20NI ONT, the settings are loaded by four scfg.txt files.
+For rewriting settings, `/userdata/scfg.txt` and `/tmp/scfg.txt` are used.
+
+- `/config/default_scfg.txt`  (ReadOnly) <br>
+Contains the manufacturer's default settings,<br>
+which are read first at ONT startup and are the lowest priority settings.<br>
+- `/config/scfg.txt` (ReadOnly)<br>
+Contains settings set by the firmware creator,<br>
+which have higher priority than default_scfg.txt and will overwrite the settings if there is a conflict.<br>
+- `/userdata/scfg.txt` (RW)<br>
+Contains settings set by the user or ISP.<br>
+It has a higher priority than /config/scfg.txt and will overwritte the settings if there is a conflict.<br>
+Can be edited and saved.<br>
+- `/tmp/scfg.txt` (Can't Save)<br>
+It contains dynamically generated settings based on values stored on its custom ROM (mtd9, mtd10)<br>
+Since they are generated on tmpfs and cannot be saved directly,<br>
+they are rewritten via the `#ONT> system/misc` command on the ONT.<br>
+
+The settings are overwritten and loaded at startup with the following priority.
+
+***(High)*** `/tmp/scfg.txt` > `/userdata/scfg.txt` > `/config/scfg.txt` > `/config/default_scfg.txt` ***(Low)***
+
+### Misc Command
+To configure settings using the MISC command, execute the following command.
+```
+#ONT> system/misc
+#ONT/system/misc>
+```
+
```

Comment:
this should be said at the beginning if I am not mistaken in the template (or in AFM0002/3 page) there is a section userful file and userful binary